apps/purchase/urls.py

```diff
@@ -18,4 +18,8 @@
     path('', include(router.urls)),
     path('purchase_order/', PurchaseOrderViewSet.as_view(), name='purchase-order-list-create'),
     path('purchase_order/<str:pk>/', PurchaseOrderViewSet.as_view(), name='purchase-order-detail-update-delete'),
+    path('purchase_invoice_orders/', PurchaseInvoiceOrderViewSet.as_view(), name='purchase-invoice-orders-list-create'),
+    path('purchase_invoice_orders/<str:pk>/', PurchaseInvoiceOrderViewSet.as_view(), name='purchase-invoice-orders-detail-update-delete'),
+    path('purchase_return_orders/', PurchaseReturnOrderViewSet.as_view(), name='purchase-return-orders-list-create'),
+    path('purchase_return_orders/<str:pk>/', PurchaseReturnOrderViewSet.as_view(), name='purchase-return-orders-detail-update-delete'),
 ]
```

Comment:
Change purchase_return_orders/ to purchase_return_order/
Change purchase_invoice_orders/ to purchase_invoice_order/